packages/lesswrong/components/common/Home.jsx

```diff
@@ -91,6 +91,17 @@ const Home = (props, context) => {
         </div>} >
         <Components.PostsList terms={recentPostsTerms} showHeader={false} />
       </Components.Section>
+      <Components.Section
+        title="Community"
+        titleLink="/community"
+        titleComponent={<Link className="events-near-you" to="/community">Events Near You</Link>}
```

Comment:
I think this should say "Find Events Near You". If it doesn't break the formatting